datacapturegallery/src/main/java/com/google/android/fhir/datacapture/gallery/utils/LanguageSwitcherUtils.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.datacapture.gallery.utils
+
+import android.app.Activity
+import android.content.Context
+import android.content.DialogInterface
+import android.content.Intent
+import android.content.res.Configuration
+import android.content.res.Resources
+import android.os.Build
+import android.os.LocaleList
+import android.widget.ArrayAdapter
+import androidx.appcompat.app.AlertDialog
+import com.google.android.fhir.datacapture.gallery.R
+import java.util.Locale
+
+object LanguageSwitcherUtils {
```

Comment:
```suggestion
object LocaleUtils {
```

prefer the name locale because it is more than languages - you could have the same language but due to where you are is displayed differently. also slightly more in line with the rest of the code.